azure-pipelines.yml

```diff
@@ -60,7 +60,29 @@ jobs:
         displayName: Copy Maestro Project Artifacts
 
       - task: PublishBuildArtifacts@1
+        displayName: Publish Service Fabric Packages Artifact
         inputs:
           PathtoPublish: $(Build.ArtifactStagingDirectory)\ServiceFabric\
-          ArtifactName: Service Fabric Packages
+          ArtifactName: ServiceFabric
+          ArtifactType: Container
+          
+      - task: PublishBuildArtifacts@1
+        displayName: Publish Darc Package Artifact
+        inputs:
+          PathtoPublish: artifacts\packages\$(_BuildConfig)\Shipping\Microsoft.DotNet.Darc.1.0.0-dev.nupkg
+          ArtifactName: Darc
+          ArtifactType: Container
+      
+      - script: $(Build.SourcesDirectory)/.dotnet/dotnet publish -o $(Build.ArtifactStagingDirectory)\Maestro.Data
+        workingDirectory: src/Maestro/Maestro.Data
+        displayName: Create Data Package
+        
+      - script: cp /y src\Maestro\update-database.ps1 $(Build.ArtifactStagingDirectory)\Maestro.Data\update-database.ps1
```

Comment:
There's a "copy task" you could use, but this is probably fine too.  You may want to fully qualify the first path here and the PublishBuildArtifact path above just for consistency

Reply:
I did look at that, but it's more about folders than files, so you have to specify a lot of parameters to get just one file, so the script seemed easier to understand. :-)

Reply:
I did specify the paths though.